machine/accton/accton_as7712_32x/machine.make

```diff
@@ -23,6 +23,9 @@ endif
 # Accton Technology Corporation IANA number
 VENDOR_ID = 259
 
+UEFI_ENABLE = yes
+PXE_EFI64_ENABLE = yes
```

Comment:
Thank you for your friendly reminder.  I will send another PR to fix the problem.